frontend/src/components/sidebar/sidebar.module.scss

```diff
@@ -0,0 +1,12 @@
+.main {
+    background-color: #106cc8;
```

Comment:
not sure what the proper way to do this is, but since this file is imported as a module it feels strange to call this class sidebar as then it will be called sidebarStyles.sidebar, so ive opted to say "main". open to other suggestions

Reply:
Hmmm yeah I get you, I'm not good at names. I've not really used css modules - I've used SCSS instead so this is a little new to me.

Reply:
Actually looking at index.tsx I think naming it sidebar is fine as it means there is no ambiguity.

Reply:
I think I'd go with something like `sidebar-container` or similar. I'll admint I've not actually seen a className declared using the `className={sidebarStyles.main}` syntax before - was this something you've done on another project?

Reply:
> Hey Guveer, mind writing a fuller description for this pull request! It helps me and Iain to properly review it! Also what is nice if you are doing front end tickets is to have a screen shot of the relevant change. Firefox has an inbuilt screenshot capture that might make this easier.

is linking the ticket sufficient, in this case I cant think of what else to add.

Reply:
> I think I'd go with something like `sidebar-container` or similar. I'll admint I've not actually seen a className declared using the `className={sidebarStyles.main}` syntax before - was this something you've done on another project?

Yeah, we did this in our grad project, the "main" part is probably not standard as I came up with it and I imagine they just let it slide. The part of having classnames in an object is using modules which is used a lot and very useful as it avoids css collisions without making classnames in your scss file longer than needed. Im happy to not use this if people prefer.

Reply:
I'm happy for you to continue to use it if worked well for you guys on the grad project 👍